source/elements/oneDAL/source/glossary.rst

```diff
@@ -114,11 +135,15 @@ Machine learning terms
         A special property of the :term:`data format`: the data are stored as one contiguous memory block.
 
     Data format
-        The representation of internal structure of the data: this information is enough to efficiently access them.
```

Comment:
we can or can't introduce data layout but we need to understand differences between two defs

Reply:
Added `data layout` term to show a difference between them